README.rst

```diff
@@ -364,124 +364,62 @@ the data in accordance with a SM-only scenario.
    :alt: postfit.png
    :align: center
 
-Local testing
-=============
-
-*Optional*
-
-If you would like to test the analysis locally (i.e. outside of the REANA
-platform), you can proceed as follows:
-
-.. code-block:: console
-
-   $ # this analysis example uses yadage; let's install it
-   $ mkvirtualenv yadage
-   $ pip install yadage==0.13.5 yadage-schemas==0.7.16 packtivity==0.10.0
-   $ # we can now run the analysis workflow
-   $ sudo yadage-run _run workflow/databkgmc.yml
-   $ # let up check output files
-   $ ls -l _run/plot/*.pdf
-   -rw-r--r-- 1 root root 19193 May 16 15:34 _run/plot/postfit.pdf
-   -rw-r--r-- 1 root root 19450 May 16 15:34 _run/plot/prefit.pdf
-
 Running the example on REANA cloud
 ==================================
 
-First we need to create a `reana.yaml <reana.yaml>`_ file describing the
-structure of our analysis with its inputs, the code, the runtime environment,
-the workflow and the expected outputs:
+We start by creating a `reana.yaml <reana.yaml>`_ file describing the above
+analysis structure with its inputs, code, runtime environment, computational
+workflow steps and expected outputs:
 
 .. code-block:: yaml
 
-   version: 0.2.0
-   inputs:
-    parameters:
-       nevents: 160000
-   outputs:
-     files:
-      - outputs/plot/postfit.pdf
-   environments:
-    - type: docker
-      image: reanahub/reana-demo-bsm-search
-   workflow:
-     type: yadage
-     file: workflow/databkgmc.yml
-
-We proceed by installing the REANA command-line client:
+    version: 0.2.0
+    inputs:
+      parameters:
+        nevents: 160000
+    workflow:
+      type: yadage
+      file: workflow/databkgmc.yml
+    outputs:
+      files:
+       - plot/postfit.pdf
 
-.. code-block:: console
+In this example we are using a Yadage workflow engine.
```

Comment:
We could remove the phrase, since the use of Yadage was already mentioned above.  (And we are using only one workflow engine in the example.)